src/components/Map.ts

```diff
@@ -154,7 +154,7 @@ export class Map extends Component<MapProps, MapState> {
         return document.getElementsByClassName("t-page-editor-iframe")[0] as HTMLIFrameElement;
     }
 
-    private onResizeIframe(_event: CustomEvent) {
+    private onResizeIframe() {
         if (this.mapLoader) {
```

Comment:
Add a check for `this.mapLoader.map` too

Reply:
Should use `this.state.loaded`